.codeclimate.yml

```diff
@@ -42,6 +42,8 @@ engines:
         enabled: false
       Rubocop/Style/IfUnlessModifier:
         enabled: false
+      Rubocop/Style/GuardClause:
+        enabled: false
```

Comment:
Disabling this for the same reason I disabled the one line if/unless statements - rubocop yells at us for trying to pragmatically keep line length down